import-service/src/libs/s3-functions.ts

```diff
@@ -0,0 +1,65 @@
+import { S3Client, DeleteObjectCommand, CopyObjectCommand } from "@aws-sdk/client-s3";
+
+const csv = require('csv-parser');
+
+const parseFile = async (S3, params) => {
+  return new Promise<void>(async (resolve, reject) => {
+    const S3Stream = S3.getObject(params).createReadStream();
+    const { Bucket, Key } = params;
+
+    S3Stream
+      .pipe(csv())
+      .on('data', data => {
```

Comment:
Here you should push data (chunk) into an array of chunks:
`chunks.push(data)`

Reply:
Why? The task only asked me to write them to the console, not to collect them for further processing.